### DISCUSSION TITLE
Audio to Mac Quicktime and AirPlay simultaneously?

### DISCUSSION THREAD

**Discussion by williemyers123** [ExistentialAudio/BlackHole#785] [General]
hello all first time here. Ive installed Blackhole on my MacBook Pro OS 12.7.4 (Monterey). I'm hoping to (a.) open a stream in FIrefox then, (b.) record that stream in Quicktime while (c.) mirroring that stream to an AirPlay TV. I tried it and the AirPlay looked/sounded fine, but teh QT recoring had no audio?

**Comment by devinroth** [comment]
What exactly did you try?

Devin
